app/lib/fake_data/complaints.rb

```diff
@@ -0,0 +1,72 @@
+require "ffaker"
+
+module FakeData::Complaints
+  def self.complaint_data
+    {
+      id: FFaker::Random.rand(9999),
+      attributes: {
+        issueLastUpdated: FFaker::Time.date,
+        creationDate: FFaker::Time.date,
+        closedDate: FFaker::Time.date,
+        reopenedDate: FFaker::Time.date,
+        initialContactDate: FFaker::Time.date,
+        type: FFaker::Random.rand(2),
+        otherType: FFaker::Lorem.word,
+        issue: FFaker::Lorem.phrase,
+        priority: FFaker::Random.rand(3),
+        status: FFaker::Random.rand(3),
+        dueDate: FFaker::Time.date,
+        # these just sound cool for use as the grantee name :)
+        grantee: FFaker::AddressUK.street_name
+      },
+      relationships: {
+        grantAward: {
+          meta: {
+            id: FFaker::Random.rand(9999)
+          }
+        },
+        grantProgram: {
+          meta: {
+            id: FFaker::Random.rand(999)
+          }
+        },
+        region: {
+          meta: {
+            id: FFaker::Random.rand(10)
+          }
+        }
+      },
+      links: {
+        self: "TODO",
```

Comment:
[thought] should these be valid URLs? something like `https://example.com/TODO` just in case someone at some point tries parsing it?

Reply:
Works for me!